test/integration/blockchain-events.spec.ts

```diff
@@ -69,7 +71,7 @@ describe('Blockchain Strategy', function () {
         expect(await isPinned(app.ipfsProvider!, file.cid)).to.be.false()
 
         // Creates Agreement with funds only for one period
-        await createAgreement(app, file, 1, file.size * 10)
```

Comment:
I think this can stay the same? `file.size` is in MB already, and we want only one period, hence the `* 10`...